dtfs-central-api/src/services/repositories/utilisation-reports-repo/utilisation-reports-repo.test.ts

```diff
@@ -78,10 +94,58 @@ describe('utilisation-reports-repo', () => {
     it('sorts the data by year then month', async () => {
       // Arrange
       const bankId = MOCK_UTILISATION_REPORT.bank.id;
-      const report1 = { ...MOCK_UTILISATION_REPORT, month: 2, year: 2022 };
-      const report2 = { ...MOCK_UTILISATION_REPORT, month: 3, year: 2021 };
-      const report3 = { ...MOCK_UTILISATION_REPORT, month: 1, year: 2022 };
-      const report4 = { ...MOCK_UTILISATION_REPORT, month: 2, year: 2021 };
```

Comment:
Would be nice to keep these on one line as it makes the test a lot easier to reason about. Could we add a function just below
```
describe('getUtilisationReportDetailsByBankId', () => {
```
as follows:
```
const getMockReport = ({ bankId, year, month }: { bankId: string; year: number; month: number }): UtilisationReport => ({
  ...MOCK_UTILISATION_REPORT,
  bank: {
    ...MOCK_UTILISATION_REPORT.bank,
    id: bankId,
  },
  reportPeriod: {
    start: { month, year },
    end: { month, year },
  },
});
```
then I think we can update this to:
```
const report1 = getMockReport({ bankId, month: 2, year: 2022 });
const report2 = getMockReport({ bankId, month: 3, year: 2021 });
const report3 = getMockReport({ bankId, month: 1, year: 2022 });
const report4 = getMockReport({ bankId, month: 2, year: 2021 });
```

Reply:
It doesn't appear that `bankId` is a field which needs to be passed in as this test uses the same value as `MOCK_UTILISATION_REPORT` anyway